Sources/MuxUploadSDK/InputInspection/UploadInputInspector.swift

```diff
@@ -115,7 +115,7 @@ class AVFoundationUploadInputInspector: UploadInputInspector {
                             formatDescription
                         )
 
-                        if max(videoDimensions.width, videoDimensions.height) > 1920 {
+                        if max(videoDimensions.width, videoDimensions.height) > 3840 {
```

Comment:
possible correctness issue: If a user selected a max resolution of 720p or 1080p and the video was larger than that, would `nonStandardReasons` still contain `.videoResolution`?